articles/why-node-js-is-popular/index.md

```diff
@@ -0,0 +1,90 @@
+
+### How and why did Node.js gain popularity?
+Node.js was introduced into the developer world in 2009 by the initial writer Ryan Dahl. Back in 2009, Apache HTTP server was a popular web server. But, it had a major problem, it could not handle a lot of concurrent connections. When Node.js was created it was able to solve the Apache problem. It offered scalability and parallel code execution. It introduced a new logic and approach for JavaScript server-side development. This has helped many developers.
+
+Since then its popularity has wildly grown. For instance in [Github](https://github.com/nodejs/node), Node.js has 71.9k stars, 17.6k forks, and 2.9k watchers. In [Stackshare](https://stackshare.io/nodejs) it has 54.4k followers and 8.2k votes. These numbers alone can show how popular Node.js is. Popular tech giants like Microsoft and Netflix use Node.js.
+
+For the second year in a row, Node.js won the top spot in the StackOverflow's [2020 developer survey](https://insights.stackoverflow.com/survey/2020#technology-other-frameworks-libraries-and-tools-all-respondents3). Over half of the respondents in the survey reported having used it in their projects.
+
+![Half of the respondents use Node.js](/engineering-education/why-node-js-is-popular/stackoverflow1.jpg)
+[Image Source](https://insights.stackoverflow.com/survey/2020#technology-other-frameworks-libraries-and-tools-all-respondents3)
+
+Node.js took the first position as the most wanted technology in the survey.
+
+![most wanted technology 2020](/engineering-education/why-node-js-is-popular/stackoverflow2.jpg)
+[Image Source](https://insights.stackoverflow.com/survey/2020#technology-other-frameworks-libraries-and-tools-all-respondents3)
+
+According to Node.js [metrics](https://nodejs.org/metrics/), Node.js has been downloaded more than 1 billion times globally.
+
+### Why Node.js is popular among developers.
+
+#### 1. It is easy to learn Node.js.
+This is one of the major contributors to its wide usage.
+Among many developers, JavaScript skills are mandatory. Node.js uses JavaScript. So, learning Node.js is relatively easy and few weeks of learning can get you up and running. Without JavaScript Knowledge, learning Node.js will take a bit longer. Therefore, learning Node.js is not that tricky.
+
+#### 2. The scalability offered.
+Node.js is designed with scalable network applications in mind. Node.js can handle many concurrent requests. This is the main reason it quickly became popular among developers and large companies. With the ability to handle many simultaneous requests, the server does not strain. A callback is fired only after a request, but if there are no more jobs to be done, Node.js will go to sleep. Node.js is only online when needed. This avoids unnecessary usage of your RAM.
```

Comment:
`With the ability to handle many simultaneous requests, the server does not strain.`

`Can handle many simultaneous requests without straining the server` sounds better.